src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,22 @@
+.catalog {
+  &__wrapper {
+    display: flex;
+    flex-wrap: wrap;
+    margin: 0 auto;
+    max-width: 1024px;
```

Comment:
In this task you need to always have 4 blocks on the page, but when I reduce your page, I see less than 4 blocks
(there should always be 4 cards in a row (not 2, 3 or 5))

Reply:
but I don't know, maybe it's not critical

Reply:
yeah, that's becouse i used max-width for make adaptive container
but in this case it's bad idea.
fixed. ty